content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -94,6 +94,7 @@ different Kubernetes components.
 | `TokenRequest` | `false` | Alpha | 1.10 | |
 | `VolumeScheduling` | `false` | Alpha | 1.9 | 1.9 |
 | `VolumeScheduling` | `true` | Beta | 1.10 | |
+| `ScheduleDaemonSetPods` | `false` | Alpha | 1.11 | |
```

Comment:
There are two `ScheduleDaemonSetPods` in this table. Did you revert the doc change for this feature in 1.10?

Reply:
Removed; not sure how this happened. The old PR was closed instead of merged.